app/models/metric.rb

```diff
@@ -22,6 +24,12 @@
 #
 
 class Metric < ApplicationRecord
+  enum interval: { daily: 'daily', weekly: 'weekly', monthly: 'monthly', all_times: 'all_times' }
+  enum name: { review_turnaround: 'review_turnaround' }
+
   belongs_to :metrics_definition, inverse_of: :metrics
   belongs_to :ownable, polymorphic: true
+
+  validates :interval, inclusion: { in: intervals.keys }
+  validates :name, inclusion: { in: names.keys }
```

Comment:
It's so cool that with Postgres enums you get db level constraint-like validations for free